README.md

```diff
@@ -1,2 +1,180 @@
+
+<div align="center">
+  <img src="Astroboy-removebg.png" alt="logo" width="120"  height="auto" />
+  <br/>
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+> **Hello Microverse** is a starter exercise to create new projects based on specific settings and requirements to ensure proper structure and best practices. By cloning this repo and running `npm install` we will have a basic boilerplate with properly configured linters and correct initial file structrure.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> This boilerplate focuses only on `linters` and `.gitignore`. No JS or Server are included in this repo.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3.org/html/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">SCSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.netlify.com/">Netlify</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Linters**: [Lighthouse](https://developer.chrome.com/docs/lighthouse/overview/), [Webhint](https://webhint.io/), and [Stylelint](https://stylelint.io/.)
+- **A `.github/workflows/linters.yml`** file to run linters on GitHub Pull Request.
+- **A** `.gitignore` file, including `node_modules/`.
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://hello-microverse.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> If you are a new developer and would like to have a basic project boilerplate to start practicing your code or creating your first app-features, cloning this project will give you what you need!
+> To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Have [npm](https://www.npmjs.com/package/npm) installed in your project:
+  ```sh
+  curl -qL https://www.npmjs.com/install.sh | sh
+  ```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+git clone https://github.com/astroboyReloaded/Hello-Microverse.git
```

Comment:
- [OPTIONAL] Always ensure your README file reflects what is in your project. This link is not todo-list link. Always try to pay attention to details. This is very important as a developer. Kindly make your title reflect your project title.